docs/dev_guide/newcomers.rst

```diff
@@ -150,30 +150,28 @@ Next we will want to setup the conda environment and we will need to add the `co
 .. code:: bash
 
     $ conda config --add channels conda-forge
-    $ conda create -n sunpy-dev sunpy
+    $ conda create -n sunpy-dev pip
     $ conda activate sunpy-dev
 
-This will create a new conda environment called "sunpy-dev" and install the latest version of SunPy from the conda-forge channel.
-The next step is remove the conda version of SunPy and install the development version of SunPy.
+This will create a new conda environment called "sunpy-dev" and install the latest version of pip from the conda-forge channel.
+The next step is get a developement version of sunpy.
 This will require that `git`_ be installed.
+If you have a `GitHub`_ account, we suggest that you `fork`_ the `sunpy repository`_ (the fork button is to the top right) and **use that url for the clone step**.
+This will make submitting changes easier in the long term for you:
 
 .. warning::
 
     Do not clone the sunpy repository into ``$HOME/sunpy``. Depending on the operating system this location is used to store downloaded data files.
     This will cause conflicts later on, so the last argument (``sunpy-git``) on the ``git clone`` line will become the local folder name of the cloned repository.
-
-.. note::
-    If you have a `GitHub`_ account, we suggest that you `fork`_ the `SunPy repository`_ (the fork button is to the top right) and **use that url for the clone step** below.
-    This will make submitting changes easier in the long term for you:
+    Otherwise you are free to clone to any other location.
 
 .. code:: bash
 
-    $ conda remove sunpy
-    $ git clone https://github.com/sunpy/sunpy.git sunpy-git
```

Comment:
The reason this was the sunpy url was to allow people who don't want to create a github account to participate.